examples/private-endpoint/privateEndpoint.json

```diff
@@ -28,41 +28,67 @@
   },
   "Mappings": {},
   "Resources": {
-    "PrivateEndpoint": {
-      "Type": "MongoDB::Atlas::PrivateEndpoint",
+    "AtlasPrivateEndpointService": {
+      "Type": "MongoDB::Atlas::PrivateEndpointService",
```

Comment:
should we leave the private endpoint example as it was, and create a new one using new resources? will also be helpful for users to compare.